src/app/theme/tamagui.config.ts

```diff
@@ -10,7 +10,7 @@ import {
   tokens as baseTokens,
 } from '@tamagui/theme-base';
 
-import colors from './colors';
+import colors from '@app/shared/lib/constants/colors';
```

Comment:
Please make it importable like this:
`import { colors } from '@app/shared/lib';`